stage1/main.tf

```diff
@@ -26,23 +26,23 @@ resource "kubernetes_namespace_v1" "dmtr" {
 }
 
 module "aws_elb_controller" {
-  source       = "../../modules/aws/elb-controller/stage1"
-  count        = var.cloud_provider == "aws" ? 1 : 0
+  source       = "../modules/aws/elb-controller/stage1"
+  count        = var.cloud_provider == "aws" || var.cloud_provider == "gcp" ? 1 : 0
```

Comment:
GCP doesn't need the AWS load balancer controller. It ships with a working load balancer controller.